constraint_utils/constraint_basics.py

```diff
@@ -685,14 +695,23 @@ def annotate_constraint_groupings(ht: Union[hl.Table, hl.MatrixTable],
 # Model building
 def build_coverage_model(coverage_ht: hl.Table) -> (float, float):
     """
-    Calibrates coverage model (returns intercept and slope)
+    Calibrates coverage model (returns intercept and slope).
+    
+    :param coverage_ht: Low coverage Table with observed variants for each population and possible variants.
+    
+    :return: Tuple with intercept and slope.
     """
     return tuple(coverage_ht.aggregate(hl.agg.linreg(coverage_ht.low_coverage_obs_exp, [1, coverage_ht.log_coverage])).beta)
 
 
 def build_plateau_models(ht: hl.Table, weighted: bool = False) -> Dict[str, Tuple[float, float]]:
     """
-    Calibrates high coverage model (returns intercept and slope)
+    Calibrates high coverage model (returns intercept and slope).
```

Comment:
This returns a dictionary because it makes 2 different models, you need to point out what those two models are. Also, what are all the expected annotations on HT?